File: tests/fixtures/java/ControlFlow.java
import java.util.List;

public class ControlFlow {
    public String classify(int n) {
        if (n > 0) {
            return "positive";
        } else if (n < 0) {
            return "negative";
        } else {
            return "zero";
        }
    }

    public int sumTo(int n) {
        int total = 0;
        for (int i = 0; i <= n; i++) {
            total = total + i;
        }
        return total;
    }

    public void printAll(List<String> items) {
        for (String item : items) {
            System.out.println(item);
        }
    }

    public int countDown(int start) {
        int n = start;
        while (n > 0) {
            n = n - 1;
        }
        return n;
    }
}

File: tests/fixtures/go/control_flow.go
package main

import "fmt"

func classify(n int) string {
	if n < 0 {
		return "negative"
	} else if n == 0 {
		return "zero"
	} else {
		return "positive"
	}
}

func sumTo(n int) int {
	total := 0
	for i := 0; i < n; i++ {
		total = total + i
	}
	return total
}

func printItems(items []string) {
	for _, item := range items {
		fmt.Println(item)
	}
}
